### PR TITLE
Create vet clinic database

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this Pull Request, I have :

-[x] Created vet_clinic database.

- [x] Created animals table.
- [x] Inserted data into animals' table.
- [x] Query animals table.
- [x] Screenshot of the results is attached to this PR:
Capture1
Capture2

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int NOT NULL PRIMARY KEY,
+    name varchar(100),
+    date_of_birth date,
+    escape_attempts int,
+    neutred boolean,
+    weight_kg float
 );
```
Comment (tresorsawasawa) [inline schema.sql]: - [ ] As required for the project, kindly ensure that the `weight_kg` from your `animals` table has the correct data type as it is supposed to be of type `decimal` but `numeric` data type is also accepted. Make sure to fix it

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @rachaouldbabaali 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @tresorsawasawa 😃

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @rachaouldbabaali , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Fixed bug
✅ ... Great readme

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @joskalenda 😃

**Pull request merged**